.gitlab/.gitlab-ci.yml

```diff
@@ -636,14 +668,16 @@ s3 dump:
     - |
       for goos in "darwin" "linux"; do
         for goarch in "amd64" "arm64"; do
-          for binary in "featurebase" "fbsql"; do
-            aws s3 cp ${binary}_${goos}_${goarch} s3://molecula-artifact-storage/featurebase/${CI_COMMIT_BRANCH}/${CI_COMMIT_SHORT_SHA}/${binary}_${goos}_${goarch}
-            aws s3 cp ${binary}_${goos}_${goarch} s3://molecula-artifact-storage/featurebase/${CI_COMMIT_BRANCH}/_latest/${binary}_${goos}_${goarch}
-          done
+          aws s3 cp featurebase_${goos}_${goarch} s3://molecula-artifact-storage/featurebase/${ci_commit_branch}/${ci_commit_short_sha}/featurebase_${goos}_${goarch}
```

Comment:
`${ci_commit_branch}` and `${ci_commit_short_sha}` need to be capitalized or they end up `""` (empty string)

Reply:
Crap. Sometimes I accidentally type `u` in vim while I have text highlighted and it lowercases everything. I must have done that here without realizing it.

Thanks for catching that.